include/seqan3/std/bit

```diff
@@ -9,13 +9,36 @@
 /*!\file
  * \brief Provides the C++20 \<bit\> header if it is not already available.
  * \author Enrico Seiler <enrico.seiler AT fu-berlin.de>
+ * \author Marcel Ehrhardt <marcel.ehrhardt AT fu-berlin.de>
  */
 
 #pragma once
 
 #if __has_include(<bit>)
 #include <bit>
 #endif // __has_include(<bit>)
+#include <climits>
+#include <type_traits>
+
+//!\brief This defines __cpp_lib_bitops.
+#ifndef SEQAN3_CPP_LIB_BITOPS
+#   if defined(__cpp_lib_bitops) && __cpp_lib_bitops >= 201907L
+#       define SEQAN3_CPP_LIB_BITOPS 1
+#   endif
+#endif
+
+/*!\brief This defines __cpp_lib_int_pow2.
+ * Note: g++-9.3 -std=c++2a on ubuntu 20.04 did `#define __cpp_lib_int_pow2 201806L`, which is an older implementation
+ * of the __cpp_lib_int_pow2 proposal.
```

Comment:
Does this mean they implemented the old version, or they set the macro to the wrong value?
Seems to be the latter case, but it reads like the former case :)

Reply:
They implemented the first revision of the proposal, which defined `__cpp_lib_int_pow2` as `201806L`, but we expect `__cpp_lib_int_pow2` to be the latest revision which has the value `202002L`.

Reply:
```suggestion
 * Note: g++-9.3 -std=c++2a on ubuntu 20.04 implemented `#define __cpp_lib_int_pow2 201806L`, which is an older version
 * of the __cpp_lib_int_pow2 proposal.
```

(does this exceed line length?)